docs/dev/table/sqlClient.md

```diff
@@ -456,8 +456,6 @@ catalogs:
 
 Currently Flink supports two types of catalog - `FlinkInMemoryCatalog` and `HiveCatalog`.
 
-For more information about catalog, see [Catalogs]({{ site.baseurl }}/dev/table/catalog.html).
```

Comment:
please revert this line. catalog doc is going to be added in https://github.com/apache/flink/pull/8976

Reply:
If #8976 would be merged before Flink-1.9 release, I would wait for your PR merged and rebase code then.

Reply:
yes. it's gonna take a while though

Reply:
If it's gonna be a while then we should merge the PR as is and add this line in #8976.

Reply:
As discussed with @bowenli86 offline, we would first let this PR merged.

Reply:
+1 for making this in asap since it's a blocker for 1.9.0 release.